README.md

```diff
@@ -21,18 +21,40 @@ We're working on [tutorials](https://napari.org/tutorials/), but you can also qu
 
 ## installation
 
-**napari** can be installed on most macOS, Linux, and Windows systems with Python 3.6, 3.7 and 3.8 by calling
+### from pip, with "batteries included"
+
+napari can be installed on most macOS, Linux, and Windows systems with
+Python 3.6, 3.7 and 3.8 using pip:
 
 ```sh
-$ pip install napari
+pip install napari[all]
 ```
 
+napari needs a library called [Qt](https://www.qt.io/) to run its user interface
+(UI). In Python, there are two alternative libraries to run this, called
+[PyQt5](https://www.riverbankcomputing.com/software/pyqt/download5) and
+[PySide2](https://doc.qt.io/qtforpython/). By default, we don't choose for you,
+and simply running `pip install napari` will not install either. You *might*
+already have one of them installed in your environment, thanks to other
+scientific packages such as Spyder or matplotlib. If neither is available,
+running napari will result in an error message.
```

Comment:
"running napari will result in an error message asking you to install one of them".